### DISCUSSION TITLE
[Question] Spacing per col for v5

### DISCUSSION THREAD

**Discussion by lenamtl** [twbs/bootstrap#33774] [Q&A] [answered]
Hi,
Using Bootstrap v5
How can I set a spacing mt-3 to the second column only when column is col-md?
Or is there a better method

```
<div class="row mb-3">	
	<div class="col-lg-8 col-md-12"></div>	
	<div class="col-lg-4 col-md-12"></div>
</div>
```

**Comment by Hiws** [comment, marked as answer]
The spacing utilities support breakpoints for when the utility should be used.
<https://getbootstrap.com/docs/5.0/utilities/spacing/#notation>

In your case you'd add `mt-md-3` to add a top margin when on the md breakpoint.
The issue is that this will also be valid on the `lg` breakpoint, which is why you'll also need to add `mt-lg-0` to set the margin back to 0.


I haven't used Bootstrap 5 yet, but perhaps there's a smarter way to add spacing between columns when they collapse in v5, but the above should be the answer to your question.

**Reply by lenamtl** [reply]
Thanks

**Reply by hwang1588** [reply]
thanks!

**Comment by lenamtl** [comment]
Hi,

I just want to say that it's not working for **xs**
`mt-xs-3` 

so we need to use 
`mt-3`

This is a bit confusing, but this is working fine 
I have combined 
`mt-lg-0 mt-md-3 mt-sm-3 mt-3`

**Reply by Hiws** [reply]
You don't need the `mt-sm-3` and `mt-md-3` classes.
The classes work for the specified breakpoint and up, unless overwritten. So you only need to apply them if you want to overwrite it with a different value, like `mt-lg-0`
